### PR TITLE
Introduce Voter stub

### PR DESCRIPTION
**Pull request by VincentLanglet**
Does such a stub require a test ?

### CODE REVIEW

**Comment by ondrejmirtes** [comment]
I'm not sure that everyone who `extends Voter` wants this functionality. In case they do, try introducing these generics into Symfony directly.

**Comment by VincentLanglet** [comment]
> I'm not sure that everyone who `extends Voter` wants this functionality.

Same could be said for almost every stub I would say.
For instance I get lot of trouble with this stub https://github.com/phpstan/phpstan-symfony/issues/322

I don't consider this stub as an hack or a wrong stub.
And I thought this was one of the purpose of `skipCheckGenericClasses`, people can skip this stub.
Or just `Voter<string, mixed>` will do nothing.

> In case they do, try introducing these generics into Symfony directly.

Symfony is a kinda slow adopter about new annotations as shown in https://github.com/symfony/symfony/pull/46613
Also, they only accepts such things on new branch, which means it would be only available on Sf 6.3.

But I can start by this to get feedback and decide after for phpstan-symfony, sure. 👍 
Edit: opened https://github.com/symfony/symfony/pull/49033

**Comment by VincentLanglet** [comment]
> I'm not sure that everyone who extends Voter wants this functionality. In case they do, try introducing these generics into Symfony directly.

Hi @ondrejmirtes, the template was accepted by symfony in 6.3 https://github.com/symfony/symfony/pull/49033
Do we merge this to add the template in older symfony version ?

File: stubs/Symfony/Component/Security/Core/Authorization/Voter/Voter.stub [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * @template TSubject of mixed
+ */
+abstract class Voter implements VoterInterface
+{
+    /**
+     * Determines if the attribute and subject are supported by this voter.
+     *
+     * @param mixed $subject
+     *
+	 * @phpstan-assert-if-true TSubject $subject
```
Comment (ondrejmirtes) [inline stubs/Symfony/Component/Security/Core/Authorization/Voter/Voter.stub]: Wrong formatting

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: stubs/Symfony/Component/Security/Core/Authorization/Voter/Voter.stub (hunk 1 above)
Comment (VincentLanglet) [reply stubs/Symfony/Component/Security/Core/Authorization/Voter/Voter.stub]: Nice catch, fixed.

**Review by ondrejmirtes — APPROVED** [review]

**Comment by ondrejmirtes** [comment]
Thank you.

**Pull request merged**